submission/Amitkumar345-2020-09-26.ipynb

```diff
@@ -0,0 +1,679 @@
+        "       [ 7,  8,  9, 10, 11, 12],\n",
+        "       [13, 14, 15, 16, 17, 18],\n",
+        "       [19, 20, 21, 22, 23, 24],\n",
+        "       [25, 26, 27, 28, 29, 30]])`"
+      ]
+    },
+    {
+      "cell_type": "code",
+      "metadata": {
+        "id": "WqH6lUS19vin"
+      },
+      "source": [
+        "npArray = np.arange(1,31).reshape(5,6)"
+      ],
+      "execution_count": 101,
+      "outputs": []
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "v6GnL0-a9viq"
+      },
+      "source": [
+        "#### Create a Numpy 1d array consisting of a random number between 0 and 1\n",
+        "\n",
+        "`array([0.42829726])`"
+      ]
+    },
+    {
+      "cell_type": "code",
+      "metadata": {
+        "id": "Bysy4Y0Z9viq"
+      },
+      "source": [
+        "npArray = np.random.rand(1)"
+      ],
+      "execution_count": 100,
+      "outputs": []
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "Psqi0iKJ9viu"
+      },
+      "source": [
+        "#### Create a Numpy 1d array consisting of six *random* numbers between 0 and 1\n",
+        "\n",
+        "`array([0.48986417, 0.61869789, 0.83943457, 0.94889576, 0.79561515, 0.28859058])`"
+      ]
+    },
+    {
+      "cell_type": "code",
+      "metadata": {
+        "id": "NQzitn6I9viv"
+      },
+      "source": [
+        "npArray = np.random.rand(6)"
+      ],
+      "execution_count": 99,
+      "outputs": []
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "sh1A43Wr9vix"
+      },
+      "source": [
+        "#### Create a Numpy 2d array of shape (2,3) consisting of five *random* numbers between 0 and 1\n",
+        "\n",
+        "`array([[0.3991418 , 0.62631826, 0.82763635],\n",
+        "       [0.9232742 , 0.08517427, 0.20155819]])`"
+      ]
+    },
+    {
+      "cell_type": "code",
+      "metadata": {
+        "id": "eWSKaOtV9viy"
+      },
+      "source": [
+        "npArray = np.random.rand(6).reshape(2,3)"
+      ],
+      "execution_count": 98,
+      "outputs": []
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "9RPZPS2t9vi2"
+      },
+      "source": [
+        "#### Create a Numpy 1d array consisting of 25 random numbers sampled from a *Standard Normal Distribution*\n",
+        "\n",
+        "`array([-0.01804177, -0.11468092, -0.4790744 , -0.44706856,  0.49891779,\n",
+        "        1.53806364, -1.03689237,  0.04379123, -0.6659274 , -0.54294319,\n",
+        "       -0.08016372, -0.64441992, -0.12417271,  0.89901988, -0.08695241,\n",
+        "        0.23540321,  1.77803936,  1.39912268,  0.19886053, -0.56546787,\n",
+        "        0.09538678, -0.47930312,  2.15532489,  0.62643712,  1.19941788])`"
+      ]
+    },
+    {
+      "cell_type": "code",
+      "metadata": {
+        "id": "cRkc58PE9vi3"
+      },
+      "source": [
+        "random = np.random.normal(0,1,25)"
```

Comment:
Create a Numpy 1d array consisting of 25 random numbers sampled from a Standard Normal Distribution